src/Orc.FileAssociation/Services/FileAssociationService.cs

```diff
@@ -102,5 +104,15 @@ public async Task UndoAssociationFilesWithApplicationAsync(ApplicationInfo appli
             }
 
         }
+
+        public async Task OpenPropertiesWindowForExtensionAsync(string extension)
+        {
+            var appPath = AppDomain.CurrentDomain.BaseDirectory;
```

Comment:
Am not sure about this, it's probably wise to provide another overload with application path parameter